README.md

```diff
@@ -1,2 +1,5 @@
 # slack-invite
 Flask app to invite on slack
+
+You don't need to do any change in these files. 
```

Comment:
Maybe add that you can change the bg image if you like would be nice, what do you think @annanda ?